bigchaindb/db/utils.py

```diff
@@ -29,7 +29,6 @@ def create_database(conn, dbname):
     logger.info('Create database `%s`.', dbname)
     r.db_create(dbname).run(conn)
 
-
```

Comment:
Could you please put the blank line back? There should be two blank lines between functions as per [PEP 8](https://www.python.org/dev/peps/pep-0008/#blank-lines)

> Surround top-level function and class definitions with two blank lines.